app/overrides/hostgroup_play_roles.rb

```diff
@@ -0,0 +1,16 @@
+# Displays Ansible roles tab on Hostgroup form
+Deface::Override.new(
+  :virtual_path => 'hostgroups/index',
+  :name => 'ansible_row_title',
+  :insert_after => "erb[loud]:contains('Actions')",
+  :partial => 'foreman_ansible/ansible_roles/ansible_row_title',
+  :original => '016f2f86bd199275e7f3bebef77fefa90dd218f1'
```

Comment:
Fair enough - honestly I normally remove them as they change so often, but if you'll be on the lookout for the alerts, that's a valid reason